.github/workflows/docker.yaml

```
@@ -79,6 +81,7 @@ jobs:
             --label "github.run_id=${{ github.run_id }}" \
             -f functions.Dockerfile \
             .
+            docker tag abbotimages.azurecr.io/aseriousbiz/abbot-py/runner-py-functions:$IMAGE_VERSION abbotimages.azurecr.io/aseriousbiz/abbot-py/runner-py-functions:$COMMIT_LABEL
```

Bring over `env.IMAGE_BASE_NAME` for symmetry with JS workflow?

---

It actually _had_ that variable originally, but this repo builds two images (`runner-py` and `runner-py-functions`) so I took it out. Eventually the other repos will too

---

Why not `IMAGE_BASE_NAME = abbotimages.azurecr.io/aseriousbiz/abbot-py`?

```suggestion
            docker tag $IMAGE_BASE_NAME/runner-py-functions:$IMAGE_VERSION $IMAGE_BASE_NAME/runner-py-functions:$COMMIT_LABEL
```